### DISCUSSION TITLE
View example in swagger ui

### DISCUSSION THREAD

**Discussion by ezeparziale** [fastapi/fastapi#10211] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from pydantic import BaseModel, ConfigDict, Field


class Team(BaseModel):
    model_config = ConfigDict(from_attributes=True)

    id: int = Field(description="ID of team", examples=[1])
    name: str = Field(
        description="Team name",
        examples=["test"],
    )


# Fastapi
from fastapi import FastAPI

app = FastAPI()


@app.post("/")
async def root(team: Team) -> "Team":
    new_team = {"id": team.id, "name": team.name}
    return new_team
```

### Description

Finally I upgrade all my apis to fastapi 0.103.1 with pydantic v2
I use `openapi examples` and I have similar swagger ui except in schema where example dont appear.

v0.98.0
![Captura de pantalla 2023-09-05 a la(s) 16 19 19](https://github.com/tiangolo/fastapi/assets/82482391/446c8a5b-af81-4166-ab02-bd279dcd27fc)

v0.103.1
![Captura de pantalla 2023-09-05 a la(s) 16 20 33](https://github.com/tiangolo/fastapi/assets/82482391/28f52502-80aa-4153-975e-036a4efab2d4)

I see all the examples but in the `openapi.json`, but in the swagger ui not, I miss some parameter ?

### Operating System

Windows, macOS

### FastAPI Version

0.103.1

### Pydantic Version

2.3.0

### Python Version

3.11

**Comment by jack-t-lange** [comment]
I think this is more of a `pydantic` question, but hopefully I can help. Try adding the `json_schema_extra` attribute to your model_config.

https://fastapi.tiangolo.com/tutorial/schema-extra-example/

 ```
 model_config = ConfigDict(
        from_attributes=True,
        json_schema_extra={
            "example": {
                "id": "1234",
                "name": "Foo",
            }
        },
    )
```

**Comment by YuriiMotov** [comment]
Initial code works well in current version (fastapi==0.103.1, pydantic==2.3.0)
